extensions/Worker.Extensions.CosmosDB/src/CosmosDBConverter.cs

```diff
@@ -80,57 +82,52 @@ private CosmosDBInputAttribute GetBindingDataContent(ModelBindingData bindingDat
             Type _ when targetType == typeof(CosmosClient) => CreateCosmosClient<CosmosClient>(cosmosAttribute),
             Type _ when targetType == typeof(Database) => CreateCosmosClient<Database>(cosmosAttribute),
             Type _ when targetType == typeof(Container) => CreateCosmosClient<Container>(cosmosAttribute),
-            _ => await CreateTargetObjectAsync(targetType, cosmosAttribute)
+            _ => ConvertObject(await CreateTargetObjectAsync(targetType, cosmosAttribute), targetType)
         };
 
-        private async Task<object> CreateTargetObjectAsync(Type targetType, CosmosDBInputAttribute cosmosAttribute)
+        private object ConvertObject(object data, Type targetType)
```

Comment:
Moving to `Stream` will require using stream-specific JSON apis, we cannot treat it as object. But that is simple, can make this change:

``` CSharp
private async Task<object?> ConvertObjectAsync(Stream stream, Type targetType)
{
    using (stream) // I think we need to dispose stream when we are done with it.
    {
        return await JsonSerializer.DeserializeAsync(stream, targetType, _serializerOptions);
    }
}
```

Reply:
We definitely do need to dispose, thanks for the reminder